modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -71,8 +69,7 @@ def get_appointment(appointment_id)
         with_monitoring do
           response = perform(:get, get_appointment_base_path(appointment_id), params, headers)
           appointment = response.body[:data]
-          prepare_appointment(appointment)
-          extract_appointment_fields(appointment)
+          prepare_appointment(appointment, { facilities: true, clinics: true })
```

Comment:
Previously, GET appointments with specified ID always populates facilities and clinic details via the controller logic so I've kept it the same here. This is an open question whether we want to let params set this behavior like is done with the GET appointments with no specified ID.

Reply:
We discussed this during our AVS conversations. This is the expected behavior.